### DISCUSSION TITLE
When is the next meeting?

### DISCUSSION THREAD

**Discussion by gobengo** [omigroup/omigroup#20] [Q&A] [answered]
Just poked around to make sure I don’t miss the next meeting. Maybe it’s not scheduled yet?

If it is scheduled, will someone link me to information about dateline?
If it’s not scheduled, will someone advise on how I can find out when meetings gets scheduled?

ty!

**Comment by robertlong** [comment, marked as answer]
We're trying to schedule it now. You can join the conversation here: https://github.com/omigroup/OMI/discussions/10#discussioncomment-661582

I think we need to schedule it by tomorrow afternoon at the latest.

**Reply by mrmetaverse** [reply]
We are looking at Wednesday 5/5/21 at 6AM PDT and 6PM PDT for a two-part OKR workshop. General meetings should begin after we've ratified our OKRs. 

Maybe we have a little get together tomorrow too? Like a social hour in hubs for anyone who can make it?

**Comment by mrmetaverse** [comment]
answered
